app/controllers/leidinggevendenbeheer/bestuursfunctie/functionarissen/new-person.js

```diff
@@ -0,0 +1,38 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+export default class LeidinggevendenbeheerBestuursfunctieFunctionarissenNewPersonController extends Controller {
```

Comment:
i think we may not need this anymore as creating persons will be dealt with by the person selector component in the form